cedar-policy-cli/src/lib.rs

```diff
@@ -495,33 +495,21 @@ pub fn validate(args: &ValidateArgs) -> CedarExitCode {
     let validator = Validator::new(schema);
     let result = validator.validate(&pset, mode);
 
-    let exit_code = if !result.validation_passed()
+    if !result.validation_passed()
```

Comment:
usually I prefer switching the `then` and `else` branches in order to not have `if !cond`

Reply:
I don't understand what you're suggesting here? Negate the whole:
`!result.validation_passed() || (args.deny_warnings && !result.validation_passed_without_warnings())` and swap the `then` and `else` branches? IMO `result.validation_passed() && (!args.deny_warnings || result.validation_passed_without_warnings())` isn't any nicer.

Reply:
Missed that the condition was more complicated.  Ignore me, sorry